repository/SmalltalkCI-Coverage-Core.package/SCIStdoutCoverageWriter.class/instance/export.in..st

```diff
@@ -0,0 +1,10 @@
+exporting
+export: codeCoverage in: projectDirectory
+
+	| coveragePercentage |
+	self printCoverageDetails: codeCoverage.
+	
+	coveragePercentage := codeCoverage coveredMethods size / codeCoverage includedMethods size * 100.
```

Comment:
This could allowdivide by zero
ToDo: Write test

Reply:
To avoid that, you could use
```smalltalk
codeCoverage coveredMethods size / (codeCoverage includedMethods size max: 1) * 100
```

Reply:
Fixed in 83f8a5f50f27596511c4f714aac098ff0818d45a